front/src/modules/ui/tooltip/__stories__/Tooltip.stories.tsx

```diff
@@ -0,0 +1,62 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import { userEvent, within } from '@storybook/testing-library';
+
+import { CatalogDecorator } from '~/testing/decorators/CatalogDecorator';
+import { ComponentDecorator } from '~/testing/decorators/ComponentDecorator';
+
+import { AppTooltip, TooltipValues } from '../AppTooltip';
+
+const meta: Meta<typeof AppTooltip> = {
+  title: 'UI/Tooltip/Tooltip',
+  component: AppTooltip,
+};
+
+export default meta;
+type Story = StoryObj<typeof AppTooltip>;
+
+export const Default: Story = {
+  args: {
+    place: 'bottom',
+    content: 'Tooltip Test',
+    anchorSelect: '#hover-text',
```

Comment:
I don't think we need anchorSelect if we use parameters: { hover } (see comment below)

Reply:
Hey @charlesBochet, Unlike the Chip which listens for the `hover` event, the Tooltip does not hence there's no pseudo state we can force on the tooltip to make it appear.

Reply:
Hey @charlesBochet ,

tl;dw
`react-tooltip` does not actually listen for the `hover` state, it listens for `mouseenter` and `mouseleave` events. There's no pseudo class we can apply on a component to for the tooltip to show up. For this particular use-case, emulating a user hover(using `play`) shows the tooltip because it involves having a mouse pointer enter the actual component(in the background, I think this is what happens). However, this too will not work  on certain browsers(e.g edge).

Uploading pseudo states on tooltip.mp4.mp4…

Reply:
ok, sounds good :)